cookbook/form/direct_submit.rst

```diff
@@ -28,7 +28,7 @@ submissions::
             return $this->redirect($this->generateUrl('task_success'));
         }
 
-        return $this->render('AcmeTaskBundle:Default:new.html.twig', array(
+        return $this->render('AppBundle:Default:new.html.twig', array(
```

Comment:
this needs further fixing, as templates live in `app/Resources/views` with the best practices. This means you get `default/new.html.twig` (new standard is to use lowercase template paths).

This doesn't have to be fixed in this PR though, using AppBundle consistently is already a huge step forward.